index.ts

```diff
@@ -0,0 +1,4 @@
+import * as deployment from './scripts/deploy';
+import * as types from './typechain-types';
+
+export { types, deployment };
```

Comment:
sorry this was my fault. I just did some testing and we need to change this whole file with this:

```suggestion
export * from './typechain-types'; 
export * from './scripts/deploy';
```

Reply:
well or depends, why do we need to export the deploy scripts? Are we going to use outside contracts?

Reply:
@franciscotobar why do we want to change it? what's the issue exactly?

Reply:
By keeping it in that way, we are exporting it like this:

`
import {
  deployment,
  types
} from '@rsksmart/rif-relay-contracts';
`

So to access to each typing we need to do:
`types.DeployVerifier__factory`

Reply:
Before it was working because we were accessing to the src files and now since the src is not included, we cannot access like we where doing it before, by doing that change can do this:

`import {
  DeployVerifier__factory,
} from '@rsksmart/rif-relay-contracts';`

Reply:
@franciscotobar good point, thank you for pointing it out 🙏🏽